tests/test_execute.py

```diff
@@ -113,6 +113,53 @@ def test_execute_notebook_profile_runtime(cells, tmp_empty):
     assert Image(filename="out-runtime.png")
 
 
+@pytest.mark.parametrize(
+    "profile_memory,profile_runtime",
+    [
+        ["mem.png", "time.png"],
+        ["bogus", "time1.png"],
+        ["mem2.png", True],
+        [True, False],
```

Comment:
Let's see what happens when it's `[True, True]`

Reply:
Added 👍🏽